_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -109324,11 +109324,15 @@
 /area/medical/morgue)
 "dCC" = (
 /turf/closed/wall/r_wall,
-/area/maintenance/department/medical/morgue)
+/area/medical/genetics{
+	name = "Cryogenic Storage"
+	})
```

Comment:
(I think there's a)Problem here, you made it just a renamed genetics instead of just it's own area. This makes it so atmos and power are linked between genetics and the cryo storage. I can make it if ya need me to

Reply:
Yeah this needs its own area.

Reply:
done